frontend/src/App.js

```diff
@@ -48,97 +59,87 @@ function AppProvider(props) {
 function AppRouter(props) {
   return (
     <Switch>
-      <PrivateRoute path="/dashboard"
-        exact={true}
-        component={Dashboard} />
-
-      <PrivateRoute path="/dashboard/create-course"
-        exact={true}
-        component={CreateNewCourse} />
-
-      <PrivateRoute path="/profile"
-        exact={true}
-        component={EmployeeProfile} />
-
-      <PrivateRoute path="/user" 
-        exact={true}
-        component={UserList}>
-      </PrivateRoute>
-
-      <Route path="/taskslist" 
-        exact={true}
-        component={TaskList}>
-      </Route>
 
-      <Route path="/taskslist/createtask" 
+      // Homepage Routes
+      <Route path="/"
         exact={true}
-        component={CreateTaskGlobal}>
+        component={HomePage}>
       </Route>
-
-      <Route path="/statistics" 
+      <Route path="/aboutus"
         exact={true}
-        component={Statistics}>
+        component={AboutUs}>
       </Route>
 
+      // Login Profile User Routes
       <Route path="/login"
         exact={true}
         component={LogIn}>
       </Route>
-
       <Route path="/signup"
         exact={true}
         component={SignUp}>
       </Route>
-
       <Route path="/signout"
         exact={true}
         component={SignOut}>
       </Route>
-
-      <Route path="/task"
+      <PrivateRoute path="/profile"
         exact={true}
-        component={TasksList}>
-      </Route>
-
-      <Route path="/task/:taskId"
+        component={EmployeeProfile} />
+      <PrivateRoute path="/user" 
         exact={true}
-        component={ViewTask}>
-      </Route>
+        component={UserList}>
+      </PrivateRoute>
 
-      <Route path="/recipelist/:recipeId"
+      // Dashboard Routes
+      <PrivateRoute path="/dashboard"
         exact={true}
-        component={ViewRecipe}>
-      </Route>
+        component={Dashboard} />
+      <PrivateRoute path="/dashboard/create-course"
+        exact={true}
+        component={CreateNewCourse} />
+      <PrivateRoute path="/dashboard/view-training"
+        exact={true}
+        component={ViewTrainingPage} />  
```

Comment:
Can you address everything in the PR you closed #87

Reply:
those changes are being done in the that branch will push as soon as its done

Reply:
OK next time branch off master please, not uncompleted changes.